SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeploy.java

```diff
@@ -89,6 +89,7 @@
   private final Optional<Long> healthcheckMaxTotalTimeoutSeconds;
 
   private final Optional<HealthcheckOptions> healthcheck;
+  private final Optional<String> healthCheckResultFilePath;
```

Comment:
I think we can make this part of the `HealthcheckOptions` object. There are healthcheck-related fields outside that object, but in general they are deprecated, in favor of the ones consolidated in `HealthcheckOptions`